expression/builtin_math_vec.go

```diff
@@ -409,6 +409,21 @@ func (b *builtinPowSig) vectorized() bool {
 	return true
 }
 
+func (b *builtinAbsRealSig) vecEvalReal(input *chunk.Chunk, result *chunk.Column) error {
+	if err := b.args[0].VecEvalReal(b.ctx, input, result); err != nil {
+		return err
+	}
+	f64s := result.Float64s()
+	for i := 0; i < len(f64s); i++ {
+		f64s[i] = math.Abs(f64s[i])
```

Comment:
would it be faster if we only calculate the non NULL values?

Reply:
when only calculate the non NULL values, like that:
		if result.IsNull(i) {
			continue
		}
BenchmarkVectorizedBuiltinMathFunc/builtinAbsRealSig-VecBuiltinFunc-8         	  500000	      3210 ns/op	       0 B/op	       0 allocs/op

when calculate all values,
BenchmarkVectorizedBuiltinMathFunc/builtinAbsRealSig-VecBuiltinFunc-8         	 2000000	       956 ns/op	       0 B/op	       0 allocs/op

so, calculate is falster.

Reply:
got it